CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## main / unreleased
 
+* [CHANGE] Return Bad Request from frontend if the provided tag is invalid in SearchTagValuesV2 endpoint [#5493](https://github.com/grafana/tempo/pull/5493/) (@carles-grafana)
```

Comment:
i'd call this a bugfix. it should have always returned 400 for an invalid attr name